src/jsutils/__tests__/didYouMean-test.js

```diff
@@ -35,4 +35,27 @@ describe('didYouMean', () => {
       ' Did you mean the letter "A"?',
     );
   });
+  // Test for handling Duplications
+  it('Handle duplicates, resulting in single unique element', () => {
+    expect(didYouMean(['A', 'A'])).to.equal(' Did you mean "A"?');
+  });
+  it('Handle duplicates, resulting in two elements', () => {
+    expect(didYouMean(['A', 'B', 'A', 'B'])).to.equal(
+      ' Did you mean "A" or "B"?',
+    );
+  });
+  it('Handle duplicates, resulting in more thentwo elements', () => {
+    expect(didYouMean(['A', 'B', 'C', 'C', 'B', 'A'])).to.equal(
+      ' Did you mean "A", "B", or "C"?',
+    );
+  });
```

Comment:
This single test actually make sense so please extract it into separate PR

Reply:
there is a typo: "more thentwo" -> "more than two"

Reply:
On second thought: It's the responsibility of `suggestionList` to remove duplicates so `didYouMean` should show duplicates as is.